staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -691,9 +702,12 @@ func (c *Config) Complete(informers informers.SharedInformerFactory) CompletedCo
 		}
 		c.ExternalAddress = net.JoinHostPort(c.ExternalAddress, strconv.Itoa(port))
 	}
-
-	completeOpenAPI(c.OpenAPIConfig, c.Version)
-	completeOpenAPIV3(c.OpenAPIV3Config, c.Version)
+	var ver *version.Version
+	if c.EffectiveVersion != nil {
+		ver = c.EffectiveVersion.EmulationVersion()
+	}
+	completeOpenAPI(c.OpenAPIConfig, ver)
+	completeOpenAPIV3(c.OpenAPIV3Config, ver)
```

Comment:
This appears to have broken `c.Version`.  We can either delete `c.Version` as no longer supported (I'm ok with this) or honor it, but we cannot allow someone to continue setting an ineffective value.

Reply:
removed `c.Version`